Detections/SigninLogs/LegacyAuthAttempt.txt

```diff
@@ -0,0 +1,18 @@
+// Name: Login attempts using Legacy Auth
```

Comment:
I got over 10000 results - should this be summarized / and made a hunting rather than a detection query?

Reply:
Moved to Hunting